presto-main/src/main/java/com/facebook/presto/execution/SqlTaskExecution.java

```diff
@@ -118,16 +118,52 @@ public static SqlTaskExecution createSqlTaskExecution(
             Executor notificationExecutor,
             QueryMonitor queryMonitor)
     {
+        LocalExecutionPlan localExecutionPlan;
```

Comment:
I'd get rid of this constructor and have the caller (SqlTaskExecutionFactory?) convert {planner,fragment} -> local execution plan, since planner and fragment are not used by this class anymore.